doc/api/net.md

```diff
@@ -205,34 +308,37 @@ persist*, they are removed when the last reference to them is closed. Do not
 forget JavaScript string escaping requires paths to be specified with
 double-backslashes, such as:
 
-    net.createServer().listen(
-        path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
+```js
+net.createServer().listen(
+  path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
+```
 
-The parameter `backlog` behaves the same as in
-[`server.listen(port[, hostname][, backlog][, callback])`][`server.listen(port, host, backlog, callback)`].
+The `backlog` argument is the maximum size of the queue of pending connections.
+The maximum possible value will be determined by the operating system.
 
 ### server.listen(port[, hostname][, backlog][, callback])
 <!-- YAML
 added: v0.1.90
 -->
 
-Begin accepting connections on the specified `port` and `hostname`. If the
-`hostname` is omitted, the server will accept connections on any IPv6 address
-(`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise. Use a
-port value of `0` to have the operating system assign an available port.
+The `server.listen()` method instructs the server to begin accepting connections
+on the specified `port` and `hostname`. If the `hostname` is omitted, the server
+will accept connections on any IPv6 address (`::`) when IPv6 is available, or
+any IPv4 address (`0.0.0.0`) otherwise. Use a port value of `0` to have the
+operating system assign an available port.
 
-Backlog is the maximum length of the queue of pending connections.
-The actual length will be determined by the OS through sysctl settings such as
-`tcp_max_syn_backlog` and `somaxconn` on Linux. The default value of this
-parameter is 511 (not 512).
+The `backlog` argument is the maximum size of the queue of pending connections.
+The maximum possible value will be determined by the operating system.
 
-This function is asynchronous.  When the server has been bound,
-[`'listening'`][] event will be emitted.  The last parameter `callback`
-will be added as a listener for the [`'listening'`][] event.
+The `server.listen()` method is asynchronous. When the server has been bound,
+the [`'listening'`][] event will be emitted.
 
-One issue some users run into is getting `EADDRINUSE` errors. This means that
-another server is already running on the requested port. One way of handling this
-would be to wait a second and then try again:
+If a `callback` function is given, it will be added as a listener for the
+[`'listening'`][] event.
+
+An `EADDRINUSE` error will occur if another server is already bound to the
```

Comment:
This is a good example of text that is present here, but not in the options-based, even though it applies equally, of course.